packages/ui/client/components/Suites.vue

```diff
@@ -1,4 +1,6 @@
 <script setup lang="ts">
+import { computed } from '@vue/composition-api'
```

Comment:
It should be auto imported

Reply:
https://github.com/zhangenming/vitest/pull/3
Why is there an error here

Reply:
https://github.com/zhangenming/vitest/pull/4
and There are no errors in this one...
How is this different... thanks